src/components/header.tsx

```diff
@@ -1,11 +1,39 @@
 import { st, classes } from './header.st.css';
 import stylableLogo from '../assets/stylable.svg';
+import { useEffect, useState } from 'react';
 
 export interface HeaderProps {
     className?: string;
 }
 
 export const Header: React.FC<HeaderProps> = ({ className }) => {
+    const [isCopied, setIsCopied] = useState(false)
+
+    const onShareClick = async () => {
+        const url = window.location.href;
+
+        if ('share' in navigator) {
+            await navigator.share({
+                url,
+                title: 'Stylable Playground',
+                text: 'A code playground for testing out Stylable'
+            })
+        } else if ('clipboard' in navigator) {
+            await navigator.clipboard.writeText(url)
+            setIsCopied(true)
+        } else {
+            prompt(url)
+        }
+    }
+
+    useEffect(() => {
+        if (isCopied) {
+            setTimeout(() => {
```

Comment:
setTimeout in react effect without clear is risky.
also if this logic is UI only it might be better to solve it with simple animation (onClick).